pom.xml

```diff
@@ -398,18 +399,42 @@
                 <version>${project.version}</version>
             </dependency>
             
+            <dependency>
+                <groupId>org.slf4j</groupId>
+                <artifactId>slf4j-api</artifactId>
+                <version>${slf4j.version}</version>
+            </dependency>
+
             <dependency>
                 <groupId>org.junit</groupId>
                 <artifactId>junit-bom</artifactId>
                 <version>5.9.1</version>
                 <type>pom</type>
                 <scope>import</scope>
             </dependency>
+            
+
+            <dependency>
+                <groupId>org.jboss.arquillian</groupId>
+                <artifactId>arquillian-bom</artifactId>
+                <version>1.7.0.Alpha10</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
 
+            <dependency>
+                <groupId>org.junit</groupId>
+                <artifactId>junit-bom</artifactId>
+                <version>5.9.1</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
             <dependency>
                 <groupId>javatest</groupId>
                 <artifactId>javatest</artifactId>
-                <version>${javatest.version}</version>
+                <version>1.0</version>
+                <scope>system</scope>
+                <systemPath>${basedir}/../lib/javatest.jar</systemPath>
```

Comment:
`${basedir}` depends of the path where you run the build.
would be better/easier to not touch this.
just have document the `mvn install:file ....` cli

Reply:
Do you recommend to remove refering to javatest jar from lib/ and to do mvn install manually for now ?

Reply:
> Do you recommend to remove refering to javatest jar from lib/ and to do mvn install manually for now ?

correct.
leave the jar in lib for now (we will remove it for sure! ;) ).
Temporary until we are in refactoring mode:
```
mvn -ntp install:install-file -Dfile=./lib/javatest.jar -DgroupId=javatest -DartifactId=javatest -Dversion=5.0 -Dpackaging=jar
```
can be in any Jenkinsfile or GHA


maven output says 
```
[WARNING] 'dependencyManagement.dependencies.dependency.systemPath' for javatest:javatest:jar should not point at files within the project directory, ${basedir}/../lib/javatest.jar will be unresolvable by dependent projects @ jakartatck:project:10.0.0-SNAPSHOT, /Volumes/workspace/dev/sources/open-sources/jetty/jakartaee-tck/pom.xml, line 437, column 29
```

this means consumers of this pom will have issues...